pyiron/atomistics/structure/atoms.py

```diff
@@ -1553,6 +1580,10 @@ def plot3d(
             vector_color (numpy.ndarray): Colors for the vectors (only available with vector_field). (Default is None,
                 vectors are colored by their direction.)
             magnetic_moments (bool): Plot magnetic moments as 'scalar_field' or 'vector_field'.
+            camera_axis (str): View the structure along 'x', 'y' or 'z' axis. (Default is 'z'.)
```

Comment:
This can be done, yes. But I think it would be nicer to have them separated. `camera_axis` specifically sets the `orientation`. The input `rotation_matrix` will need to flattened and scaled (zoom_out) to get said `orientation`.

Reply:
My arguments:

 - They do conceptually the same thing (i.e. rotation)
 - They are not compatible with each other (one will be simply ignored when the other is set)

Then I would rather say they shouldn’t coexist. I would simply set something like `orientation` instead and internally check whether it was a string or an array.

Reply:
Sound like a sane argument. Shall I change `camera_axis` to `camera_orientation`, which takes axis(str) or the rotation_matrix(3, 3 array) as input?

Reply:
sounds brilliant 👍